### ISSUE TITLE
Section on how/ high to mount lighthouses and commutator

### ISSUE THREAD

**Issue by ahleighton** [open-ephys/onix-docs#18]
Should describe what to consider, how much area the lighthouses cover and how to make sure entire setup/arena is covered, as well as pictures and part numbers for screws and frame.

**Comment by jonnew** [comment]
yes, this is a good topic. In fact, it will probably take a bit of testing to figure it out. The biggest factor is the size of the photodiode on the headstage.
